lib/brazilian_documents.ex

```diff
@@ -163,17 +163,51 @@ defmodule BrazilianDocuments do
       iex> BrazilianDocuments.valid_cnpj?(cnpj)
       true
 
+      iex> cnpj = BrazilianDocuments.generate_cnpj(subsidiary: 13)
```

Comment:
What happen if I pass a number with more than 4 digits?

```suggestion
      iex> cnpj = BrazilianDocuments.generate_cnpj(subsidiary: 12345)
```

I think we could have a test for this scenario.